delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/projections/ProjectionErrors.scala

```diff
@@ -71,6 +56,39 @@ trait ProjectionErrors {
       rcr: RemoteContextResolution
   ): Stream[Task, ServerSentEvent]
 
+  /**
+    * Return a list of errors for the given projection on a time window ordered by instant
+    *
+    * @param project
+    *   the project of the projection
+    * @param projectionId
+    *   its identifier
+    * @param timeRange
+    *   the time range to restrict on
+    * @return
+    */
```

Comment:
This comment should refer to count rather than list

These fields are clear from the name, consider removing the comments